data/infra/resources/launchd.yaml

```diff
@@ -125,45 +124,6 @@ properties_list:
   required: false
   description_list:
   - markdown: The name of the cookbook in which the source files are located.
-- property: group
-  ruby_type: String, Integer
-  required: false
-  description_list:
-  - markdown: 'When launchd is run as the root user, the group to run the job as.
-
-      If the `username` property is specified and this property is not,
-
-      this value is set to the default group for the user.'
-- property: label
-  ruby_type: String
-  required: false
-  description_list:
-  - markdown: The unique identifier for the job.
-- property: mode
-  ruby_type: Integer, String
-  required: false
-  default_value: '''0755'''
-  description_list:
-  - markdown: 'A quoted 3-5 character string that defines the octal mode. For
-
-      example: `''755''`, `''0755''`, or `00755`.'
-- property: source
-  ruby_type: String
-  required: false
-  description_list:
-  - markdown: The path to the launchd property list.
-- property: type
-  ruby_type: String
-  required: false
-  description_list:
-  - markdown: 'The type of resource. Possible values: `daemon` (default), `agent`.'
-- property: abandon_process_group
-  ruby_type: true, false
-  required: false
-  description_list:
-  - markdown: 'If a job dies, all remaining processes with the same process ID may
-
-      be kept running. Set to true to kill all remaining processes.'
```

Comment:
Should these all be deleted? They're still listed in the example code block. `label` and `mode` were just moved lower in the file.

Reply:
Oh, I see what's going on. These are duplicate properties. The dups are getting deleted here. They got added in #4004.

Reply:
This setup really is a nightmare.